parachain/pallets/ethereum-beacon-client/src/benchmarking/mod.rs

```diff
@@ -42,16 +76,19 @@ benchmarks! {
 	import_execution_header {
 		let caller: T::AccountId = whitelisted_caller();
 
-		EthereumBeaconClient::<T>::initial_sync(data::initial_sync()).unwrap();
+		let initial_sync_data = initial_sync();
+
+		EthereumBeaconClient::<T>::initial_sync(initial_sync_data.clone())?;
 
-		let sync_update: SyncCommitteePeriodUpdate<T::MaxSignatureSize, T::MaxProofBranchSize, T::MaxSyncCommitteeSize> = data::sync_committee_update();
-		SyncCommittees::<T>::set(sync_update.sync_committee_period+1, sync_update.next_sync_committee);
```

Comment:
Why did you decide to remove the sync committee update from this benchmark?

Reply:
Ah, I did not remove the sync committee update,  just use another way as following
https://github.com/Snowfork/snowbridge/blob/364c05c1896c68ad1fb257245ff529a817260234/parachain/pallets/ethereum-beacon-client/src/benchmarking/mod.rs#L85-L87